backend/src/ee/services/secret-approval-request/secret-approval-request-dal.ts

```diff
@@ -355,7 +443,7 @@ export const secretApprovalRequestDALFactory = (db: TDbClient) => {
         .from<Awaited<typeof query>[number]>("w")
         .where("w.rank", ">=", offset)
         .andWhere("w.rank", "<", offset + limit);
-      const formatedDoc = sqlNestRelationships({
+      const formattedDoc = sqlNestRelationships({
```

Comment:
So a bypasser won't be able to see the request - until they become an approver right?

I think then the other booleancheck for reviewing for secret approva needs to be like this `!isApprover || (self approval check) || canByPassCheck`